File: app/src/main/java/org/ametro/ui/navigation/adapter/IHolderFactory.java
package org.ametro.ui.navigation.adapter;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

interface IHolderFactory
{
    View createView(LayoutInflater inflater, ViewGroup parent);
    IHolder createHolder(View view);
}

File: app/src/main/java/io/github/romangolovanov/apps/ametro/ui/navigation/adapter/IHolderFactory.kt
package io.github.romangolovanov.apps.ametro.ui.navigation.adapter

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

internal interface IHolderFactory {
    fun createView(inflater: LayoutInflater, parent: ViewGroup): View
    fun createHolder(view: View): IHolder
}
